modules/ROOT/pages/_partials/install-mule-runtime-versions.adoc

```diff
@@ -0,0 +1,64 @@
+= Install Mule Runtime Engine in Studio
+
+// tag::intro[]
+Anypoint Studio bundles the latest Runtime engine version available for each release.
```

Comment:
I think it's better if you use **Mule runtime engine version** here since it is the first time you present the full name, and then you can just say **Mule version** in later instances.